pages/tidal tools/syncservers.md

```diff
@@ -19,12 +19,11 @@ date source, by continually updating your inventory on a recurring schedule.
 
 {% include note.html content="You will first need to have [Tidal Tools installed and logged in](https://guides.tidalmg.com/tidal-tools.html) to your Workspace." %}
 
-You can sync any data source with Tidal Migrations by generating a simple JSON document of the data.
+To upload your server inventory into your Tidal Migrations workspace you need to transform your data into JSON format.
 
-This data can be passed as standard input to the `tidal sync servers` command and your servers data will be
-synchronized via the API.
+This data can be passed as standard input to the `tidal sync servers` command and your servers data will be synchronized via the API.
 
-The JSON document must be created in the specified format. The top-level key must be `"servers"`,
+The JSON document must have this specified format. The top-level key must be `"servers"`,
 with a value of an array. The array can consist of the various keys as shown below, describing the server to be synced.
 You can also include any other arbitrary fields in the key `"custom_fields"`.
```

Comment:
Curious if this works as described? My understanding was that the custom fields had to be created ahead of time, or maybe I'm mistaken

Reply:
Really good point @SamDesmondKing. You are correct in that custom fields need to be created ahead of time, otherwise, it is just ignored. I will add a few more words here to make that clear.

Reply:
In the case of tidal sync servers, Tidal Tools has mechanisms to create those custom fields. The next _note_  (line 30) explaining how it works is accurate.

For example:
<img width="454" alt="image" src="https://user-images.githubusercontent.com/22829270/196725310-8bb1d8b3-d6d9-4ede-a7d7-5304110c7e5e.png">